jduong09/thirteen
# Issue 5 pt 2  game component

Resolves 90% of https://github.com/jduong09/thirteen/issues/5
Blocked by https://github.com/jduong09/thirteen/pull/9

## Summary
- Implemented pass functionality
- Implemented accepting/rejecting a combo from the player

## TODOs (future tickets will address these)
- Implement logic to validate a combo (if it truly is player 0's turn, validate that the 3 of spades is played)
- Pass back updated hand to player if a valid combo is accepted
- Redo the logic in the `<Hand>` component for clearing out the selected card. The way I did it is dumb..
- Style thing. I only did minimal styling for now while we continue to build out core functionality.

## Testing
- Pull down changes from this branch
- Start server
- For testing purposes, I've made it so that player 1 is always first (we'll clean this up later)
- Try submitting a combo (it will always be rejected unless you change `isValid` to `true`)
- Try passing turn
  - Selected cards should now be de-selected and it should be the next player's turn
